ledger/participant-integration-api/src/main/scala/platform/apiserver/ApiServerConfig.scala

```diff
@@ -39,4 +38,5 @@ case class ApiServerConfig(
     enableMutableContractStateCache: Boolean,
     maxTransactionsInMemoryFanOutBufferSize: Long,
     enableInMemoryFanOutForLedgerApi: Boolean,
+    enableErrorCodesV2: Boolean,
```

Comment:
Can we have an explicit name here, please? V2 doesn't tell the reader what this enables.
I'd suggest going with e.g. `enableSelfServiceErrorCodes` or similar.

Reply:
I like a more descriptive name. 
@tudor-da Wdyt?